src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/createbundle/CreateBundleAboutToStartEventHandler.java

```diff
@@ -130,4 +132,15 @@ private boolean isDwpResponseDocsMissing(SscsCaseData sscsCaseData) {
     private boolean hasMandatoryDocumentMissingForLegacyAppeals(List<DwpDocument> dwpResponseDocs) {
         return dwpResponseDocs.stream().anyMatch(e -> isNull(e.getValue().getDocumentLink()));
     }
+
+    private void moveDocsToDwpCollectionIfOldPattern(SscsCaseData sscsCaseData) {
```

Comment:
Aren't we already doing this in the about to submit? Why do we need to do in the about to start as well?

Reply:
Yes we are but this ticket is to stop anyone other than superuser being able to create bundle without DWP documents, if they are in the old format it incorrectly comes up with an error saying the documents aren't there.